src/Model/Profile.php

```diff
@@ -506,30 +507,40 @@ public static function getVCardHtml(array $profile, bool $block, bool $show_cont
 		return $o;
 	}
 
+	/**
+	 * Check if the event list should be displayed
+	 *
+	 * @param integer $uid
+	 * @param Mode $mode
+	 * @return boolean
+	 */
+	public static function displayEventList(int $uid, Mode $mode): bool
```

Comment:
I see what you were going for with this function name but I think it is a little ambiguous because the use of “display” as the action verb.
```suggestion
	public static function shouldDisplayEventList(int $uid, Mode $mode): bool
```

Reply:
Done